### PR TITLE
hostDisk doc

### PR DESCRIPTION
**Pull request by mfranczy**

### CODE REVIEW

**Comment by mfranczy** [comment]
@rmohr could you take a look and check if it makes sense?
@stu-gott could you check from native speaker perspective?

**Comment by mfranczy** [comment]
I forgot about PVC initialization, added to PR.

File: workloads/virtual-machines/disks-and-volumes.md [hunk 1]
```diff
@@ -376,6 +379,47 @@ spec:
...
 
+### hostDisk
+
+A ```hostDisk``` volume type provides the ability to create or use a disk image located somewhere on a node.
+It works similar to a ```hostPath``` in Kubernetes and provides two usage types:
+
+* ```DiskOrCreate``` if a disk image does not exist at a given location then create one
+* ```Disk``` a disk image must exist at a given location
+
+Example: Create a disk image with 1Gi size at /data/disk.img location and attach it to a VM.
```
Comment (stu-gott) [inline workloads/virtual-machines/disks-and-volumes.md]: Suggest rewording to:

Example: Create a 1Gi disk image located at /data/disk.img and attach it to a VM.

Comment (mfranczy) [reply workloads/virtual-machines/disks-and-volumes.md]: done
